src/components/List/List.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Book from '../Book/Book';
+
+function List() {
+  return (
+    <ul className="list">
+      <li className="list-item">
+        <Book />
+      </li>
+      <li className="list-item">
+        <Book />
+      </li>
+    </ul>
```

Comment:
- [ ] Please ensure the list component Renders a list of books using the `<Book>` component :pray:

Reply:
Done! Thanks